cupy/linalg/product.py

```diff
@@ -59,6 +60,130 @@ def vdot(a, b):
     return core.tensordot_core(a, b, None, 1, 1, a.size, ())
 
 
+def cross(a, b, axisa=-1, axisb=-1, axisc=-1, axis=None):
+    """Returns the cross product of two vectors.
+
+    The cross product of ``a`` and ``b`` in :math:`R^3` is a vector
+    perpendicular to both ``a`` and ``b``.  If ``a`` and ``b`` are arrays
+    of vectors, the vectors are defined by the last axis of ``a`` and ``b``
+    by default, and these axes can have dimensions 2 or 3.  Where the
+    dimension of either ``a`` or ``b`` is 2, the third component of the input
+    vector is assumed to be zero and the cross product calculated accordingly.
+    In cases where both input vectors have dimension 2, the z-component of
+    the cross product is returned.
+
+    Args:
+        a : cupy.ndarray
```

Comment:
The format of the docstring does not match the CuPy convention.
Please check other documented functions and fix the format.
You can also check the rendering result of documentation by yourself, following the procedure in this document: https://docs-cupy.chainer.org/en/stable/contribution.html#documentation

Reply:
I'm not able to access `_normalize_axis_index` in the built docs. How do I look for it?

Reply:
I'm not able to find `cross` in compiled docs. How do I look for it?

Reply:
Oh, I'm sorry, I forgot to mention that you need to add an entry in documentation: please add `cupy.cross` under "Matrix and vector products" section in `docs/source/reference/linalg.rst`.

Reply:
Fixed it